stock_move_line_auto_fill/tests/test_stock_picking_auto_fill.py

```diff
@@ -0,0 +1,482 @@
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.exceptions import UserError
+from odoo.tests import Form, common
```

Comment:
```suggestion
from odoo.tests import Form
from odoo.addons.base.tests.common import BaseCommon
```
Recommendation: As of Odoo 17.0, use `BaseCommon` as the standard approach.